README.md

```diff
@@ -523,6 +523,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Fidal Mathew](https://github.com/FidalMathew)
   - [Franklin Ohaegbulam](https://github.com/frankiefab100)
   - [Frazier Odhiambo](https://github.com/frazie)
+  - [Fran Pérez](https://github.com/franpersanchez)
```

Comment:
Please add your name in line 524, after Fidal Mathew.
Let us know if you need any help with that! 🙂


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

Reply:
Sure. Already done. Thanks for letting me know! I 

I hope it is fine now.